setup.py

```diff
@@ -67,7 +67,7 @@
 install_requires = [
     'bravado>=9.0.6,<10.2',
     'click>=6.7,<6.8',
-    'cwltool==1.0.20180326152342',
+    'cwltool==1.0.20180912090223',
```

Comment:
- [ ] Please update `setup.py` to declare official support for Python 3.7.

- [ ] Please amend `.travis.yaml` testing matrix as we have in `reana-cluster`:

```
matrix:
  fast_finish: true
  include:
    - python: 2.7
    - python: 3.6
    - python: 3.7
      dist: xenial
```

Reply:
Done, PR is ready.